src/rastervision/backend/tf_deeplab.py

```diff
@@ -0,0 +1,647 @@
+import os
+import glob
+import shutil
+import tarfile
+import tempfile
+import uuid
+from typing import (Dict, List, Tuple)
+from urllib.parse import urlparse
+from os.path import join
+from subprocess import Popen
+
+import numpy as np
+import tensorflow as tf
+from PIL import ImageColor
+from tensorflow.core.example.example_pb2 import Example
+from google.protobuf import (json_format)
+from object_detection.utils import dataset_util
```

Comment:
Note to make sure all TF object detection, tensorflow or deeplab imports are lazily imported so that library initialization doesn't require these to be installed (#398)

Reply:
Ok, I've got it on my TODO list.